lib/graphql/schema/input_object.rb

```diff
@@ -45,6 +50,8 @@ def initialize(values = nil, ruby_kwargs: nil, context:, defaults_used:)
             end
           end
         end
+
+        GraphQL::Execution::Lazy.all(maybe_lazies).value
```

Comment:
This is not awesome -- it forces a lazy-loading batch before the last second. Maybe this can be moved to `#prepare` or something.